app/views/organizations/index.html.erb

```diff
@@ -1,5 +1,8 @@
 <ul>
   <% @organizations.each do |org| %>
-    <li><%= link_to org.name, projects_path(organization_id: org.id) %></li>
+    <li>
+        <%= link_to org.name, projects_path(organization_id: org.id) %>, 
+        <%= link_to "dashboard", "/projects/" + org.id.to_s + "/dashboard" %>
```

Comment:
I'd like to structure this URL differently... maybe just make it an alternate param to the `index` action. (Happy to mess with it after merging... just don't set this path in stone anywhere in the meantime.)

Reply:
no worries to me